### PR TITLE
[MOB-1966] Remove Bing Experiment code

### PR DESCRIPTION
**Pull request by d4r1091**
[MOB-1966]

## Context

As the Bing Experiment conclusion, we want to remove the related code from the experiment.

## Approach

Removing references to the Bing Experiment throughout the codebase and the Test target.

[MOB-1966]: https://ecosia.atlassian.net/browse/MOB-1966?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ecotopian — DISMISSED** [review]
LGTM 🧹

File: Client/Ecosia/Experiments/Unleash/BingExperiment.swift [hunk 1]
```diff
@@ -1,50 +0,0 @@
...
-    }
-    
-    static func makeBingSearchURLFromURL(_ sourceURL: URL) -> URL? {
-        var urlComponents = URLComponents(url: sourceURL.absoluteURL, resolvingAgainstBaseURL: false)
-        urlComponents?.host = "bing.com"
-        return urlComponents?.url
-    }
-    
-    static func trackAnalytics() {
-        Analytics.shared.userSearchViaBingABTest()
```
Comment (lucaschifino) [inline Client/Ecosia/Experiments/Unleash/BingExperiment.swift:31]: Did you also remove `Analytics.userSearchViaBingABTest()`? I guess it is no longer needed and did not find the code removal change.

**Review by lucaschifino — COMMENTED** [review]
🧹 👏 

Added a couple comments to check out.

File: Client/Ecosia/Experiments/Unleash/BingExperiment.swift (hunk 1 above)
Comment (d4r1091) [reply Client/Ecosia/Experiments/Unleash/BingExperiment.swift:31]: I did not! Thanks for the reminder 🚀

File: Client.xcodeproj/project.pbxproj
[the remark this answers is not in the document]
Comment (d4r1091) [reply Client.xcodeproj/project.pbxproj:1]: I've reverted the change solely but now updated as well 💪

**Review by lucaschifino — APPROVED** [review]
LGTM 👍

**Pull request merged**